Configs/ServerSettings.md

```diff
@@ -899,11 +899,11 @@ diwako_dui_enable_compass = true;
 diwako_dui_enable_compass_dir = 1;
 diwako_dui_enable_occlusion = false;
 diwako_dui_enable_occlusion_cone = 360;
-diwako_dui_hudScaling = 2;
+force diwako_dui_hudScaling = 1;
```

Comment:
Is there a real need to force this scaling setting? would be a bit of a pain in the ass for people running higher resolutions

Reply:
This was an export of Sniper's settings, so I think it was done by accident. I will remove the forced HUD scaling.

Reply:
🚢